src/components/Molecules/NavbarLeft/NavbarLeft.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, cleanup } from '@testing-library/react';
+import NavbarLeft from './index';
+
+describe('Test NavbarLeft Component', () => {
+  afterEach(cleanup);
```

Comment:
No need to do this, [it already happens automatically](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup).

Reply:
Cool! All cleanups removed.